releasenotes/notes/cws-enable-security-profiles-by-default-647c307d94a92bd8.yaml

```diff
@@ -0,0 +1,4 @@
+---
+features:
+  - |
+    Enable CWS Security Profiles by default.
```

Comment:
```suggestion
    Enable CWS security profiles by default.
```

Reply:
@urseberry If it works for you we are gonna keep `Security Profiles` to match more closely with the product name, as can be seen here https://app.datadoghq.com/security/workload/profiles